### ISSUE TITLE
[Request]Displaying a variable in the display menus

### ISSUE THREAD

**Issue by Jojojrman** [chrismaltby/gb-studio#1311]
Right now, we can only show variables using the "display dialogue" feature. It would be awesome if we could also do this with the "Display menu" and "Display Multiple choice" options. 
And if we could use if statements in the "Display Menu" to decide whether to show options based on player conditions. That way, we can keep things tidy and only show what's relevant at the moment.

**Comment by sabra55** [comment]
can't you use if statements outside of "display menu", like "if X then display menu Y, else if Z then display menu A etc. etc."?

**Comment by Jojojrman** [comment]
Yes that is how I am doing it, it just takes up a lot more time and space that way. It would make it so much simpler if it could be directly in the menu. That way if you have a long list of things that could be there it’d simplify it down.

**Comment by pau-tomas** [comment]
The next version of GB Studio will have autocomplete for variables in single line text fields.

In the meantime, you can display variables in menu using `$000$` where `000` is the variable index. 

You can get the variable index by selecting a variable in the left sidebar and then click in the 🔽 icon of the variable editor.
